src/Apps/Order/Routes/Shipping2/__tests__/Shipping2.jest.tsx

```diff
@@ -255,7 +258,13 @@ const getAllPendingOperationNames = (env: RelayMockEnvironment) => {
 describe("Shipping", () => {
   let isCommittingMutation: boolean
   let relayEnv: MockEnvironment
-  const mockPush = jest.fn()
+  const mockPush = jest.fn(() => {
+    act(() =>
+      relayEnv.mock.getAllOperations().forEach(op => {
+        relayEnv.mock.resolve(op, MockPayloadGenerator.generate(op))
+      })
```

Comment:
I hoped this would help shorten test runs by helping pending promises bleed out quicker if the test completes an action and navigates away. It doesn't seeem to have done much.